sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -162,8 +179,14 @@ export class EventHubConsumer {
   }
 
   /**
-   * Gets an async iterator over events from the consumer.
-   * @param [options] Options to pass to the event iterator.
+   * Returns an async iterable that retrieves events.
+   *
+   * The async iterable will not indicate that it is done.
```

Comment:
"will not" or "cannot" ?

Reply:
I mean, both, but will update to `cannot`.